### PR TITLE
Update ru.js

### PR DESCRIPTION
**Pull request by AveVlad**

### CODE REVIEW

File: ru.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-	"Unread messages in %s": "Unread messages in %s",
-	"Unread messages in %s and %s": "Unread messages in %s and %s",
-	"Unread messages from %s": "Unread messages from %s",
-	"Unread messages from %s and %s": "Unread messages from %s and %s",
+	"Unread messages in %s": "Непрочитанное сообщение в %s",
+	"Unread messages in %s and %s": "Непрочитанное сообщение в %s и %s",
+	"Unread messages from %s": "Непрочитанное сообщение от %s",
+	"Unread messages from %s and %s": "Непрочитанное сообщение от %s и %s",
 	"Unread messages from %%s, %%s and one other": {
-		"one": "Unread messages from %%s, %%s and one other",
```
Comment (just-boris) [inline ru.js]: There is a suppressed third person. Here should be
```
Непрочитанные сообщения от %%s, %%s и еще кого-то
```

File: ru.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+		"other": "%d непрачитанных сообщений"
 	},
 	"Unread messages in %%s, %%s and one other": {
 		"one": "Unread messages in %%s, %%s and one other",
 		"other": "Unread messages in %%s, %%s and %d others"
 	},
-	"Unread messages in %s": "Unread messages in %s",
-	"Unread messages in %s and %s": "Unread messages in %s and %s",
-	"Unread messages from %s": "Unread messages from %s",
-	"Unread messages from %s and %s": "Unread messages from %s and %s",
```
Comment (just-boris) [inline ru.js]: Here should be 
```
Непрочитанное сообщения от %s и %s
```
because of message<b>s</b>

**Comment by cutandpastey** [comment]
@AveVlad Hey, thanks for the translations. Has there been any movement on the above comments? If so I would like to get this merged ASAP. Thanks.

File: ru.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
 {
 	"%s ago": "%s назад",
 	"%d unread message": {
-		"one": "%d unread message",
-		"other": "%d unread messages"
+		"one": "%d непрочитанное сообщение",
+		"other": "%d непрачитанных сообщений"
```
Comment (just-boris) [inline ru.js]: Typo. Should be
```
"other": "%d непрочитанных сообщений"
```
*Безударные гласные такие безударные*

**Comment by MadLittleMods** [comment]
@AveVlad Just checking in, what's your plan on this?

**Comment by AveVlad** [comment]
@MadLittleMods no, sorry

**Pull request closed without merging**